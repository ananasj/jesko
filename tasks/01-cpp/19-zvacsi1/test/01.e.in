18
15
33 93 81 50 25 25 79 100 28 52 12 61 12 84 32 72 7 14
