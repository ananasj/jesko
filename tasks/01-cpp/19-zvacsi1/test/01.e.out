48 108 96 65 40 40 94 115 43 67 27 76 27 99 47 87 22 29
